metrics/models.go

```diff
@@ -74,26 +54,18 @@ type TestId struct {
 	Name string `json:"name"`
 }
 
-type TestIdSlice []TestId
-
-func (s TestIdSlice) Len() int {
-	return len(s)
-}
+// ByTestPath sorts test ids by their test path, then name, desc.
```

Comment:
nit: Unclear whether `desc` is description or descending.

Reply:
Done.